baldrick/plugins/utils.py

```diff
@@ -0,0 +1,15 @@
+from flask import current_app
+
+__all__ = ["get_config_with_app_defaults"]
+
+def get_config_with_app_defaults(repo_handler, config_section):
```

Comment:
Can we also read from environment variables?

Reply:
It would be kinda hard to embed an arbitrary config dict in an env var?